draft-ietf-mls-protocol.md

```diff
@@ -1384,10 +1384,23 @@ enum {
     (255)
 } ContentType;
 
+enum {
+    invalid(0),
+    member(1),
+    external_authority(2),
```

Comment:
I'm not sure "authority" is quite the semantic here.  Maybe just "external" or "preconfigured"?